debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleConnectorConfig.java

```diff
@@ -401,6 +409,7 @@ public static ConfigDef configDef() {
     private final LogMiningDmlParser dmlParser;
     private final boolean archiveLogOnlyMode;
     private final boolean lobEnabled;
+    private final Set<String> logMiningUsernameExcludeList;
```

Comment:
Let's remove the "list" suffix; a) it's inconsistent with the actually used type, and b) it's not needed to encode the type in the name to begin with. I'd suggest `logMiningUserNameExcludes`.